include/tango/server/command.h

```diff
@@ -220,9 +222,10 @@ class Command
  * The object desctructor.
  */
 #ifdef HAS_UNIQUE_PTR
-    virtual ~Command() {}
+    virtual ~Command()
+    {}
 #else
-	virtual ~Command() {delete ext;}
+    virtual ~Command() {delete ext;}
```

Comment:
Actually, this was found thanks to @chedburgh who ran a tool to check cppTango code quality.
Thanks Stuart! :-)